addons/mail/controllers/thread.py

```diff
@@ -109,8 +110,9 @@ def mail_message_post(self, thread_model, thread_id, post_data, context=None, sp
             post_data["body"] = Markup(post_data["body"])  # contains HTML such as @mentions
         new_partners = []
         if "partner_emails" in kwargs:
+            additional_values = {email_normalize(email): values for email, values in kwargs.get("partner_additional_values", {}).items()}
```

Comment:
In case we have a wrong email (like "poop"), we can keep the key. Also using strict=False allows to support multi_email (aka: 'multi1@x.com, multi2@x.com' is normalized to 'multi1@x.com' as a defensive fallback to try to support multi emails)
```suggestion
            additional_values = {email_normalize(email, strict=False) or email: values for email, values in kwargs.get("partner_additional_values", {}).items()}
```